README.md

```diff
@@ -83,3 +83,22 @@ Due to [network limitation](https://docs.docker.com/docker-for-mac/networking/#k
 ```sh
 $ EXPOSE_HOST_PORTS=true make docker-testacc
 ```
+
+Releasing the Provider
+---------------------------
+
+* Create a draft of the [release](https://github.com/rancher/terraform-provider-rancher2/releases) and select create new tag for the version you are releasing
+* Create release notes by clicking `Generate release notes`
+* Copy the release notes to the CHANGELOG and update to the following format
+
+```
+# <tag version> (Month Day, Year)
+FEATURES:
+ENHANCEMENTS:
+BUG FIXES:
+```
+
+* Create a PR to update CHANGELOG
+* Copy the updated notes back to the draft release (DO NOT release with just the generated notes. Those are just a template to help you)
+* Publish the release (using the CL: git tag v0.0.0; git push -u origin v0.0.0)
+* Create an [EIO issue](https://github.com/rancherlabs/eio) for Hashicorp to publish the release
```

Comment:
Is there an expected template for the issue, or is it more of a generic "hey, we need a new release" ticket? otherwise LGTM

Reply:
There's no expected template right, it's more generic because Hashicorp has had issues before syncing the release on time. Ideally, we'd have a drone step that auto creates that ticket